uvicorn/lifespan/on.py

```diff
@@ -49,10 +50,10 @@ def __init__(self, config):
                 return
             if self.config.lifespan == "auto":
                 msg = "ASGI 'lifespan' protocol appears unsupported."
-                self.logger.info(msg)
+                self.access_logger.info(msg)
```

Comment:
I think we should use `error_logger.info(...)` for this one. (It's kinda a soft error)

Reply:
yep I was wrong